### ISSUE TITLE
Attach aria-required to Required Form Elements

### ISSUE THREAD

**Issue by DBNubs** [mautic/mautic#9145]
Would it be possible to attach the aria-required attribute to required form elements by default. At the moment we are adding this via the Attributes tab on the form element itself, however, forgetting to do so will result in this not being applied.

This would help accessibility for the visually impaired.

**Comment by RCheesley** [comment]
@DBNubs that's a great suggestion and indeed that would make forms in Mautic much better for folk with visual impairments.

Would you be interested in working on making a pull request to add this feature?

**Comment by DBNubs** [comment]
@RCheesley I would love to be able to put one together to add this. The difficulty I'm finding is where exactly in the codebase attributes like this need to be added. Any pointers would be helpful in putting this together.

**Comment by RCheesley** [comment]
Hi @DBNubs apologies for the delay in getting back to you!

In regards to the fields, each field type can be found in this folder: 

https://github.com/mautic/mautic/tree/features/app/bundles/FormBundle/Views/Field 

So, for example if adding to the 'text' field type you would edit this file: https://github.com/mautic/mautic/blob/features/app/bundles/FormBundle/Views/Field/text.html.php

If you need any help with making a PR we have the documentation here: https://contribute.mautic.org/contributing-to-mautic/developer/code/pull-requests or feel free to join the Product Team on Slack (get an invite at https://mautic.org/slack and join #t-product) and one of the team will be able to guide you!
